src/docfx/lib/schema/JsonSchemaValidator.cs

```diff
@@ -337,7 +347,7 @@ private void ValidateMicrosoftAlias(JsonSchema schema, string name, JValue scala
 
         private void ValidateDateRange(JsonSchema schema, string name, JValue scalar, DateTime date, List<Error> errors)
         {
-            var diff = date - DateTime.Now;
+            var diff = date - DateTime.UtcNow;
```

Comment:
@LowEntropyBody should this be `UtcNow` ?

Reply:
Right, UTC is better